### DISCUSSION TITLE
support for asdot notation for 4 Byte ASN

### DISCUSSION THREAD

**Discussion by empusas** [netbox-community/netbox#8237] [Ideas]
As more and more 4 Byte ASN gets assigned I find it much easier to work with asdot notation like ASN 64330.2345 as the large numbers in asplain like ASN 4215933225.
We could store only asplain in the database, but in the UI we allow to enter both notation and convert them automatically.
Also in the UI we display both values and do the convert in js.

**Comment by ryanmerolle** [comment]
seems like a feature request instead of discussion

**Comment by empusas** [comment]
sometimes it helps to read the contributing guidelines before posting

**Feature Requests**

Before filing a new feature request, consider raising your idea in a GitHub discussion first. Feedback you receive there will help validate and shape the proposed feature before filing a formal issue.

**Reply by ryanmerolle** [reply]
Feels like a decent feature to me... ha I guess I am quick to raise features myself

**Reply by jeremystretch** [reply]
Nothing wrong with that. The guide just makes the suggestion to help people form their ideas for complex features before opening a formal issue.

**Reply by empusas** [reply]
@ryanmerolle wasn´t meant as offense as it may have sound, I just tried to follow the rules.

**Reply by ryanmerolle** [reply]
> @ryanmerolle wasn´t meant as offense as it may have sound, I just tried to follow the rules.

Ha was not taken that way.  You response caused no offense.

**Comment by empusas** [comment]
I have opened a feature request now #8293. I have also give a simple example how the conversion could be done in javascript in runtime.
